src/terra/Factory/EnvironmentFactory.php

```diff
@@ -75,9 +75,11 @@ public function init($path = null)
         }
 
         try {
+            mkdir($path, 0755, TRUE);
+            chdir($path);
             $wrapper = new GitWrapper();
             $wrapper->streamOutput();
-            $wrapper->clone($this->app->repo, $path);
+            $wrapper->cloneRepository($this->app->repo . '.git', $path);
```

Comment:
Yes it absolutely would, we should probably do some sort of cli level validation of the input.